third_party/googleapis/google/api/annotations.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015, Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api;
+
+import "google/api/http.proto";
```

Comment:
I agree with @slimsag. `google/api/http.proto` looks to be more appropriate as a canonical path than `github.com/gengo/grpc-gateway/third_party/googleapis/google/api/http.proto`.

BTW, I should have added `-I$GOPATH/src/github.com/gengo/grpc-gateway/third_party/googleapis` in the example in `README.md`.